From: Antonio Terceiro <terceiro@debian.org>
Date: Sun, 1 May 2022 19:12:25 -0300
Subject: Gemfile: relax dependency on rubocop

---
 Gemfile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Gemfile b/Gemfile
index 0398a74..63c3ff5 100644
--- a/Gemfile
+++ b/Gemfile
@@ -16,6 +16,6 @@ gem "minitest", "~> 5.0"
 gem "mocha", "~> 1.2"
 
 group :development do
-  gem "rubocop", "0.82.0" # Ruby 2.4 support
+  gem "rubocop"
   gem "byebug", platform: :ruby
 end
